src/components/MultiHopTrade/types.ts

```diff
@@ -7,3 +11,41 @@ export type StepperStep = {
   content?: JSX.Element
   status?: MultiHopExecutionStatus
 }
+
+export enum SelectedQuoteStatus {
+  ReadyToPreview = 'ReadyToPreview',
+  Loading = 'Loading',
+  Updating = 'Updating',
+  SellAmountBelowMinimum = 'SellAmountBelowMinimum',
+  SellAmountBelowTradeFee = 'SellAmountBelowTradeFee',
+  InsufficientFirstHopFeeAssetBalance = 'InsufficientFirstHopFeeAssetBalance',
+  InsufficientLastHopFeeAssetBalance = 'InsufficientLastHopFeeAssetBalance',
+  InsufficientSellAssetBalance = 'InsufficientSellAssetBalance',
+  NoConnectedWallet = 'NoConnectedWallet',
+  SellAssetNotNotSupportedByWallet = 'SellAssetNotNotSupportedByWallet',
+  BuyAssetNotNotSupportedByWallet = 'BuyAssetNotNotSupportedByWallet',
+  NoReceiveAddress = 'NoReceiveAddress',
+  NoQuotesAvailableForTradePair = 'NoQuotesAvailableForTradePair',
+  NoQuotesAvailableForSellAmount = 'NoQuotesAvailableForSellAmount',
+  TradingInactiveOnSellChain = 'TradingInactiveOnSellChain',
+  TradingInactiveOnBuyChain = 'TradingInactiveOnBuyChain',
+  NoQuotesAvailable = 'NoQuotesAvailable',
+  UnknownError = 'UnknownError',
+}
+
+export type QuoteStatus = {
+  selectedQuoteErrors: SelectedQuoteStatus[]
+  quoteStatusTranslation: string | [string, InterpolationOptions]
+  quoteHasError: boolean
+  errorMessage?: string
+}
+
+export type TradeQuoteResult =
+  | {
+      isLoading: boolean
+      data: Result<TradeQuote<ThorChainId>, SwapErrorRight> | undefined
+      swapperName: SwapperName
+      error: unknown
+    } & {
+      inputOutputRatio: number
```

Comment:
```suggestion
      error: unknown
      inputOutputRatio: number
```

Reply:
Fixed in https://github.com/shapeshift/web/pull/4763/commits/af67c60008afef05b420d549558393778762aff6.